src/common.ts

```diff
@@ -118,14 +118,16 @@ declare interface CrossApp extends App {
 
 /** Get the name of an electron app for <v5 and v7< */
 export function getNameFallback(): string {
-  if (!require) {
+  let electron;
+
```

Comment:
@k15a I've got another PR in the works and this could make it into a release soon. 

Could you add some comments here to explain about esbuild and why this is required? There's always commit history but this looks like the sort of code that I would immediately clean up without a second thought!

Reply:
Hi, ESBuild has removed the warning but Evan correctly pointed out that this code was not doing anything before. If require is not defined the check will already throw a ReferenceError and your helpful error message would never reach the user. I will add a comment saying that `require` can be not defined and throwing a reference error.

<img width="471" alt="Screen Shot 2021-04-22 at 09 41 26" src="https://user-images.githubusercontent.com/20357450/115675534-f64dc300-a34e-11eb-8f2e-dcfba8bb4139.png">